embedded-graphics/src/primitives/rectangle/styled.rs

```diff
@@ -378,4 +394,56 @@ mod tests {
 
         assert_eq!(moved, Rectangle::new(Point::new(1, 2), Size::new_equal(10)));
     }
+
+    #[test]
+    fn bounding_box() {
+        let rectangle = Rectangle::new(Point::new(10, 10), Size::new(15, 20));
+
+        let base = PrimitiveStyleBuilder::new()
+            .stroke_color(BinaryColor::On)
+            .stroke_width(5);
+
+        let center = rectangle.into_styled(base.stroke_alignment(StrokeAlignment::Center).build());
+        let inside = rectangle.into_styled(base.stroke_alignment(StrokeAlignment::Inside).build());
+        let outside =
+            rectangle.into_styled(base.stroke_alignment(StrokeAlignment::Outside).build());
+
+        assert_eq!(
+            center.bounding_box(),
+            Rectangle::new(Point::new(8, 8), Size::new(19, 24)),
+            "center"
+        );
+        assert_eq!(
+            inside.bounding_box(),
+            Rectangle::new(Point::new(10, 10), Size::new(15, 20)),
+            "inside"
+        );
+        assert_eq!(
+            outside.bounding_box(),
+            Rectangle::new(Point::new(5, 5), Size::new(25, 30)),
+            "outside"
+        );
```

Comment:
Do we need these additional assertions? If you want to keep them I would at least use `offset` like in the rounded rectangle test.